build.sbt

```diff
@@ -501,6 +501,13 @@ val circe = Seq("circe-core", "circe-generic", "circe-parser")
   .map("io.circe" %% _ % circeVersion)
 val snakeyamlVersion = "2.3"
 
+// === Reactive ===============================================================
+
+val rxJavaVersion = "3.1.10"
+val rxJava = Seq(
+  "io.reactivex.rxjava3" % "rxjava" % "3.1.10"
```

Comment:
- I am surprised by usage of `rxjava`
- sure, it stands for _Reactive Java_ - but it is _not the reactive_ I was thinking about
- when talking about reactive I mean _Vue.js reactive_ - e.g.
   - [reactive core](https://vuejs.org/api/reactivity-core.html)
   - with its `ref`, `computed`, `watchEffect`, etc.
- or about [Svelte](http://wiki.apidesign.org/wiki/Svelte)'s reactivity as shown in the following video

[![Svelte Reactivity](https://img.youtube.com/vi/vwQ06AjwKkGiUKzG/0.jpg)](https://www.youtube.com/watch?v=vwQ06AjwKkGiUKzG)